modules/core/legacy/v100/genesis_test.go

```diff
@@ -14,7 +14,7 @@ import (
 	clienttypes "github.com/cosmos/ibc-go/v3/modules/core/02-client/types"
 	connectiontypes "github.com/cosmos/ibc-go/v3/modules/core/03-connection/types"
 	host "github.com/cosmos/ibc-go/v3/modules/core/24-host"
-	"github.com/cosmos/ibc-go/v3/modules/core/legacy/v100"
+	v100 "github.com/cosmos/ibc-go/v3/modules/core/legacy/v100"
```

Comment:
```suggestion
	"github.com/cosmos/ibc-go/v3/modules/core/legacy/v100"
```
?

Reply:
Go compiler is importing it like this, I guess there's `clientv100 "github.com/cosmos/ibc-go/v3/modules/core/02-client/legacy/v100"` a bit above...

Reply:
Is the go complier importing like this or is it your text editor?